modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -0,0 +1,168 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-viewing.adoc
+
+[id='nodes-nodes-viewing-listing_{context}']
+= Listing all the nodes in a cluster
+
+You can get detailed information on the nodes in the cluster.
+
+.Procedure
+
+* To list all nodes that are known to the master:
++
+----
+$ oc get nodes
+NAME                   STATUS    ROLES     AGE       VERSION
+master.example.com     Ready     master    7h        v1.9.1+a0ce1bc657
+node1.example.com      Ready     compute   7h        v1.9.1+a0ce1bc657
+node2.example.com      Ready     compute   7h        v1.9.1+a0ce1bc657
+----
+
+* To list all nodes with information on a project's pod deployment with node information
++
+----
+$ oc get nodes -o wide
+NAME                           STATUS    ROLES     AGE       VERSION           EXTERNAL-IP      OS-IMAGE                                      KERNEL-VERSION          CONTAINER-RUNTIME
+ip-172-18-0-39.ec2.internal    Ready     infra     1d        v1.10.0+b81c8f8   54.172.185.130   Red Hat Enterprise Linux Server 7.5 (Maipo)   3.10.0-862.el7.x86_64   docker://1.13.1
+ip-172-18-10-95.ec2.internal   Ready     master    1d        v1.10.0+b81c8f8   54.88.22.81      Red Hat Enterprise Linux Server 7.5 (Maipo)   3.10.0-862.el7.x86_64   docker://1.13.1
+ip-172-18-8-35.ec2.internal    Ready     compute   1d        v1.10.0+b81c8f8   34.230.50.57     Red Hat Enterprise Linux Server 7.5 (Maipo)   3.10.0-862.el7.x86_64   docker://1.13.1
+----
+
+* To list only information about a single node, replace `<node>` with the full
+node name:
++
+----
+$ oc get node <node>
+----
++
+The `STATUS` column in the output of these commands can show nodes with the
+following conditions:
++
+.Node Conditions [[node-conditions]]
+[cols="3a,8a",options="header"]
+|===
+
+|Condition |Description
+
+|`Ready`
+|The node is passing the health checks performed from the master by returning
+`StatusOK`.
```

Comment:
This is not correct.  The node reports its own readiness to the apiserver.

The Ready condition is set to false (i.e. NotReady) if one of the underlying components such as the container runtime or network is experiencing issues or is not yet configured.
https://github.com/kubernetes/kubernetes/blob/3e1c0d47c21a8afed1b93980c5aaea095c90e16a/pkg/kubelet/nodestatus/setters.go#L439